API/Controllers/SearchController.cs

```diff
@@ -0,0 +1,58 @@
+using API.DTOs;
+using API.Services.Interfaces;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[ApiController]
+[ApiVersion(1)]
+[EnableCors]
+[Authorize(Roles = "system")] // Internal access only at this time
+[AllowAnonymous]
```

Comment:
Make sure this gets removed. I am pretty sure this bypasses the above `[Authorize(Roles = "system")]`